fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/FedoraToOCFLObjectIndexImpl.java

```diff
@@ -35,10 +42,30 @@
  */
 @Component
 public class FedoraToOCFLObjectIndexImpl implements FedoraToOCFLObjectIndex {
+
     private static Logger LOGGER = LoggerFactory.getLogger(FedoraToOCFLObjectIndexImpl.class);
 
     private Map<String, FedoraOCFLMapping> fedoraOCFLMappingMap = Collections.synchronizedMap(new HashMap<>());
 
+    /**
+     * Constructor
+     * @throws IOException If we can't access the file.
+     */
+    public FedoraToOCFLObjectIndexImpl() throws IOException {
+        if (FEDORA_TO_OCFL_INDEX_FILE.exists() && FEDORA_TO_OCFL_INDEX_FILE.canRead()) {
+            Files.lines(FEDORA_TO_OCFL_INDEX_FILE.toPath()).filter(l -> {
+                final String m = l.split("\t")[0];
+                return (!fedoraOCFLMappingMap.containsKey(m));
+            }).forEach(l -> {
+                final String[] map = l.split("\t");
+                if (map.length == 3) {
+                    final FedoraOCFLMapping ocflMapping = new FedoraOCFLMapping(map[1], map[2]);
+                    fedoraOCFLMappingMap.put(map[0], ocflMapping);
+                }
+            });
```

Comment:
Possibly add a logging statement in the case that `map.length` does not equal `3`?